features/bevragen/persoon/verblijfstitel/dev/verblijfstitel-gba.feature

```diff
@@ -93,3 +120,15 @@ Achtergrond:
     | aanduiding.omschrijving | Onbekend |
     | datumIngang             | 00000000 |
     | datumEinde              | 00000000 |
+
+  Scenario: vervallen verblijfstitel (aanduiding 98) wordt niet geleverd
+    Gegeven de persoon met burgerservicenummer '000000176' heeft de volgende 'verblijfstitel' gegevens
+    | aanduiding verblijfstitel (39.10) | datum einde verblijfstitel (39.20) | datum ingang verblijfstitel (39.30) | aanduiding in onderzoek (83.10) | datum ingang onderzoek (83.20) |
+    | 98                                | 20660201                           | 20210315                            | 100000                          | 20230127                       |
```

Comment:
Is er een reden / is het dan handig om ook in onderzoek onderdeel te maken van dit scenario?

Reply:
ik wil checken dat geen enkel gegeven van verblijfstitel geleverd wordt wanneer aanduiding=98. Dus niet alleen de aanduiding, maar alles van de verblijfstitel. Dit is wat er nu niet goed werkt, nu wordt alleen de aanduiding niet geleverd, maar de rest wel. Waaruit m.i. blijkt dat het toch nuttig is dit aan de test toe te voegen